tests/tests_sbd_needs_atb_while_atb_disabled.yml

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure disabled ATB doesn't conflict with SBD
+  hosts: all
+  vars_files: vars/main.yml
+  vars:
+    ha_cluster_sbd_enabled: yes
+    ha_cluster_quorum:
+      options:
+        - name: auto_tie_breaker
+          value: 0
+
+  tasks:
+    - block:
+        - name: Set up test environment
+          include_tasks: tasks/setup_test.yml
+
+        - name: Set up test environment for SBD
+          include_tasks: tasks/setup_sbd.yml
+
+        - name: Set SBD devices variable
+          set_fact:
+            ha_cluster:
+              sbd_devices: []
+
+        - block:
+            - name: Run the role
+              include_role:
+                name: linux-system-roles.ha_cluster
+          rescue:
+            - name: Check errors
+              assert:
+                that: "{{
```

Comment:
you can use
```
that: ansible_failed_result.msg ==
        'Cannot set auto_tie_breaker to disabled when SBD needs it ' ~
        'to be enabled'
```
`~` is the jinja string concat operator - https://jinja.palletsprojects.com/en/2.9.x/templates/#other-operators

Reply:
Yes, that's definitely better, fixed. Thanks!